shared/fixes/ansible/accounts_maximum_age_login_defs.yml

```diff
@@ -3,11 +3,13 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
+- (xccdf-var var_accounts_maximum_age_login_defs)
+
 - name: Set Password Maximum Age
   lineinfile:
-      create: yes
```

Comment:
Is there any reason to fix the indentation in this PR?

Reply:
Yeah, I fixed it because I could and I was already changing these files and lines